bin/nanocld/src/services/process/stats.rs

```diff
@@ -63,3 +63,43 @@ pub async fn stats_processes(
       ),
   )
 }
+
+/// Get stats of all processes of key name
```

Comment:
The comment is wrong we only get stats of a single process

Reply:
Fixed the comment